src/main/kotlin/io/snyk/plugin/cli/CliVulnerabilitiesForFile.kt

```diff
@@ -0,0 +1,20 @@
+package io.snyk.plugin.cli
+
+class CliVulnerabilitiesForFile {
```

Comment:
Just a suggestion: maybe change it to FileCliVulnerabilities or CliFileVulnerabilities?